execution_workflows/PAQR/.gitignore

```diff
@@ -0,0 +1,3 @@
+results*/
+*log*
+.snakemake/
```

Comment:
Maybe add the downloaded scripts to gitignore? That is, `workflow/scripts_paqr` and `workflow/scripts_tpas`. 
Though I'm not sure what the best approach is.